src/routes/email/index.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import EmailController from '../../controllers/EmailController';
+
+const router = express.Router();
+
+router.post('/send-email', EmailController.sendEmail);
```

Comment:
The base path for these routes should be in the makeRoutes.js file so we can see all route paths in a single location. Also, please make the path /fullfilments/send-email

Reply:
done